docs/vendor/gcp/pipeline_run.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: demo-pipelinerun-
+spec:
+  serviceAccountName: my-ksa
+  pipelineRef:
+    resolver: git
+    params:
+      - name: url
+        value: https://github.com/tektoncd/chains.git
+      - name: pathInRepo
+        value: pipeline.yaml
+      - name: revision
+        value: main
+  workspaces:
+  - name: shared-data
+    volumeClaimTemplate:
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Mi
+  params:
+  - name: repo-url
+    value: https://github.com/bendory/mini-true.git
```

Comment:
Prefer something under tekton control so we can easily update if needed (ideally in this folder).